### ISSUE TITLE
Decreased Triceps Skinfold thickness 

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5893]
**Preferred term label:** Decreased Triceps Skinfold thickness 

**Synonyms** Decreased TSF

**Definition (free text, please give PubMed ID)** Skinfold measured halfway down the back of the upper arm is decreased. This is a way to estimate total body fat percentage., PMID: 27669294

**Parent term (use hpo.jax.org/app)** Decreased adipose tissue HP:0040063

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term

**Comment by JillMiller** [comment]
Hi Peter. I found the increased TSF on the GitHub website but not the decreased. Do you know the number assigned to this term?
